llvm/docs/CommandGuide/lit.rst

```diff
@@ -151,7 +151,12 @@ EXECUTION OPTIONS
  feature that can be used to conditionally disable (or expect failure in)
  certain tests.
 
-.. option:: --time-tests
+.. option:: --no-time-tests
+
+ Disable tracking the wall time individual tests take to execute. This is useful
+ for external build systems to orchestrate the scheduled tests.
```

Comment:
That second sentence doesn't read quite right to me from a grammatical perspective. Perhaps "This allows external build systems to orchestrate ..."?